### ISSUE TITLE
Doesn't work on 1.5. Is the project abandoned?

### ISSUE THREAD

**Issue by itsjohnthehero** [cipherxof/MGSFPSUnlock#12]
Is this going to be updated?

**Comment by cipherxof** [comment]
Yes I will update it. Someone is also free to make a pull request.

**Comment by Savemypeanuts** [comment]
Thank you, bro. I've gotten myself a 144Hz monitor recently and would love
to experience it. 👌 Much obliged.

**Comment by ShizCalev** [comment]
i've updated it for 2.0.1. download is on the pr page here https://github.com/cipherxof/MGSFPSUnlock/pull/19
